environment/src/main/resources/application.yml

```diff
@@ -90,7 +90,8 @@ environment:
   enabledParentPlatforms: AWS, MOCK
   enabledChildPlatforms: YARN, MOCK
   tunnel:
-    default: DIRECT
+    default: CLUSTER_PROXY
```

Comment:
I'm still confused about whether CLUSTER_PROXY is the default or not. Maybe this gets changed to be the default in the production deployment.

Reply:
with this change default is changed to CLUSTER_PROXY.

Reply:
All of these settings could potentially be changed via the actual deployment scripts.

Also - I know this changes the default to CLUSTER_PROXY. That is a big enough change to be made via a separate jira, instead of as part of an otherwise big patch.

Reply:
agree. i was planning to exclude this in the merge to main and create seperate commit for thta.